acquia/acquia_cms
# ACMS-1554: Updated embedded video media template to support DAM.

**Motivation**
Fixes #ACMS-1554

**Proposed changes**
Updated embedded video media template of siste studio to support DAM

**Alternatives considered**
NA

**Testing steps**
- Install site with ACMS-1554 branch
-- `composer install`
-- `./vendor/biin/acms site:install`
- Enable acquia_cms_dam module
-- `./vendor/bin/acms in acquia_cms_dam -y`
- Login into site ad admin
- head towards /node/add
- Click on Article
- Head towards body fields
- select insert from media library
- add video from dam
- verify video render on frontend

**Merge requirements**
